apiserver/service/src/main/java/com/zipline/service/contract/dto/request/ContractRequestDTO.java

```diff
@@ -97,4 +97,11 @@ public void validateProperty() {
 			throw new ContractException(ContractErrorCode.PROPERTY_REQUIRED);
 		}
 	}
+
+	public void validateDistinctParties() {
+		if (lessorOrSellerUid != null && lesseeOrBuyerUid != null && lessorOrSellerUid.equals(lesseeOrBuyerUid)) {
```

Comment:
```suggestion
		if (lessorOrSellerUid != null && lessorOrSellerUid.equals(lesseeOrBuyerUid)) {
```
[nit] 뒤에서 equlas로 검사하니까 둘 중 하나만 남겨놔도 될 것 같아요!